### PR TITLE
Minor: remove browser compat from css cookbook split nav

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/openwebdocs/project/issues/184.

### CODE REVIEW

File: files/en-us/web/css/layout_cookbook/split_navigation/index.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ browser-compat: css.properties.flex
 
 {{CSSRef}}
 
-A navigation pattern where one or more elements are separated from the rest of the navigation items.
+The \*_split navigation_ is a navigation pattern where one or more elements are separated from the rest of the navigation items.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/layout_cookbook/split_navigation/index.md:10]: I don't really get what the asterisk is for. Also, isn't a more common pattern on MDN to bold important items when first introduced?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle one small quibble

**Review by chrisdavidmills — APPROVED** [review]
Oops, meant to approve.

File: files/en-us/web/css/layout_cookbook/split_navigation/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/css/layout_cookbook/split_navigation/index.md:10]: I was asking myself the same question:
```suggestion
The **split navigation** is a navigation pattern where one or more elements are separated from the rest of the navigation items.
```

**Comment by estelle** [comment]
That was a typo. Meant to put 2 asterisks. Thanks!

**Pull request merged**